### ISSUE TITLE
Using directive to provide example value requires nested string

### ISSUE THREAD

**Issue by kylebjordahl** [anvilco/spectaql#461]
### What we expected to work:

```graphql
displayName: String! @spectaql(options: [{ key: "example", value: "Captain Planet" }])
```
But sadly this yields a syntax error: 
```
undefined:1
Captain Planet
^

SyntaxError: Unexpected token C in JSON at position 0
    at JSON.parse (<anonymous>:null:null)
    at Object.example (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:26:26)
    at /home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:64:51
    at Array.reduce (<anonymous>:null:null)
    at processDirective (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:62:92)
    at configHandler (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:91:20)
    at MapperKind.OBJECT_FIELD (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:129:5)
    at mapFields (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/@graphql-tools/utils/cjs/mapSchema.js:144:37)
    at mapSchema (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/@graphql-tools/utils/cjs/mapSchema.js:10:37)
    at transformer (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/directive.js:163:35)
    at loadSchemaFromSDLFile (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/graphql-loaders.js:103:12)
    at buildSchemas (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/build-schemas.js:44:47)
    at run (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/spectaql/index.js:55:78)
    at loadData (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/index.js:524:10)
    at run (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/dist/index.js:319:59)
    at Object.<anonymous> (/home/vscode/.npm/_npx/30d8ff16066afd73/node_modules/spectaql/bin/spectaql.js:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

### What actually works
```graphql
#                                                                  👇 note the escaped string
displayName: String! @spectaql(options: [{ key: "example", value: "\"Captain Planet\"" }])
```

(making this issue to link to a PR for a fix)

**Comment by newhouse** [comment]
This was known to me at the time I released it...but I probably should have (a) documented it better or (b) fixed it.

**Comment by newhouse** [comment]
This is fixed and released in `main` now and v`1.5.3`.

**Comment by kylebjordahl** [comment]
Thanks @newhouse !
